src/kvilib/system/KviEnvironment.cpp

```diff
@@ -56,22 +56,19 @@ namespace KviEnvironment
 		}
 		return true;
 #else
-	bool setVariable(const char *, const char *)
-	{
 		// no setenv, no putenv.. what the hell of system is this ?
 		return false;
 #endif
 #endif
 	}
 
-#ifdef HAVE_UNSETENV
-	void unsetVariable(const char * name)
+	void unsetVariable(QString szName)
```

Comment:
The `QString`s passed should be passed as `const QString&` to avoid an unnecessary copy.